packages/shared/src/components/errors/SquadLoading.tsx

```diff
@@ -1,35 +1,45 @@
-import React, { ReactElement } from 'react';
+import React, { HTMLAttributes, ReactElement } from 'react';
+import classNames from 'classnames';
 import classed from '../../lib/classed';
+import { FlexRow } from '../utilities';
 
-const PlaceholderElement = classed('div', 'bg-theme-float');
+const PlaceholderElement = classed('div', 'bg-theme-bg-tertiary');
```

Comment:
Notice the change in the background, the accurate one is supposed to be P80 which is our `bg-tertiary`.

Reply:
Just to verify do we have this issue in any other placeholders now?

Reply:
Oh, good point. Let me check if there was any.